back-end/aws-cloud9-api/src/main/java/com/serverless/dal/Question.java

```diff
@@ -0,0 +1,246 @@
+package com.serverless.dal;
+
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.fasterxml.jackson.databind.JsonNode;
+
+
+public class Question {
+
+	
+	protected String id;
+    protected String type;
+    protected String question;
+    protected String form_membership;
+    protected Integer answer;
+    protected List<String> answer_list;
+    protected Integer language;
+    protected String recruiter_id;
+
+    private Logger logger = LogManager.getLogger(this.getClass());
+    
+    public Question() {};
+    
+    public Question(JsonNode body) {
+    logger.info("Call Question constructor(JsonNode)");
+    if (body.get("type").asText().equals("0")) {
+    	this.id = body.get("id").asText();
+    	this.type = body.get("type").asText();
+    	this.question = body.get("name").asText();
+    	this.answer = body.get("answer").asInt();
+    	this.recruiter_id = body.get("recruiter_id").asText();
+    	this.language = body.get("language").asInt();
+    	this.form_membership = body.get("form_id").asText();
+    }
+    	
+    if (body.get("type").asText().equals("1")) {
+    	this.id = body.get("id").asText();
+    	this.type = body.get("type").asText();
+    	this.question = body.get("name").asText();
+    	this.answer = body.get("answer").asInt();
+    	this.recruiter_id = body.get("recruiter_id").asText();
+    	this.language = body.get("language").asInt();
+    	this.form_membership = body.get("form_id").asText();
+    	String answers = body.get("answer_list").toString(); //do zmiany
+    	logger.info("answers in string"+answers);
+    	this.answer_list =QuestionDBTable.singleChars(answers);
+    	logger.info("answers in answer_list"+answer_list);  	
+    }
+    	
+    if (body.get("type").asText().equals("2")) {
+    	this.id = body.get("id").asText();
+    	this.type = body.get("type").asText();
+    	this.question = body.get("name").asText();
+    	this.recruiter_id = body.get("recruiter_id").asText();
+    	this.language = body.get("language").asInt();
+    	this.form_membership = body.get("form_id").asText();
+
+    }
+    	
+    };
```

Comment:
Ja bym to podzielił na mniejsze funkcje np set<type>QuestionAttributes, albo create... i w tej funkcji tylko switch z wywołaniem ich.